src/SFA.DAS.ApprenticeCommitments/Data/ApprenticeRepository.cs

```diff
@@ -1,28 +1,36 @@
-﻿using System;
-using System.Threading.Tasks;
+﻿using Microsoft.EntityFrameworkCore;
 using SFA.DAS.ApprenticeCommitments.Data.Models;
 using SFA.DAS.ApprenticeCommitments.Map;
 using SFA.DAS.ApprenticeCommitments.Models;
+using System.Net.Mail;
+using System.Threading.Tasks;
 
 namespace SFA.DAS.ApprenticeCommitments.Data
 {
     public class ApprenticeRepository : IApprenticeRepository
     {
-        private readonly Lazy<ApprenticeCommitmentsDbContext> _dbContext;
+        private readonly ApprenticeCommitmentsDbContext _db;
```

Comment:
I think the reason we have Lazy<> here is to ensure the DbContext is opened within the UnitOfWork, otherwise we may have be updating the ServiceBus and SQL db on two different connections

Reply:
I think I know why you changed this. The tests would have failed. But this would be because the NSB outboxes aren't in the test database because tehy don#t belong to the EF context